### PR TITLE
Generalize the audit container filtering

### PR DESCRIPTION
**Pull request by acoburn**
Resolves: https://jira.duraspace.org/browse/FCREPO-2320

### CODE REVIEW

File: fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java [hunk 1]
```diff
@@ -129,4 +135,17 @@ public void configure() throws Exception {
                 "{{serialization.binaries}}${headers[CamelSerializationPath]}")
             .to("exec:rm");
     }
+
+    private List<Predicate> getUriFilter() {
```
Comment (birkland) [inline fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java]: This same `getUriFilter()` code appears to be repeated in four places - would it be reasonable to make it a static method on  ___, e.g. like `getUriFilter(CamelContext cxt)`?

Comment (acoburn) [inline fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java]: we could do that, but what would that common package be? It seems like creating a new shared artifact would add more complexity than the duplication of the code here. Personally, I don't like this filter at all, but I'm hesitant about removing it -- the better solution is to inject a small route in between the fedora message stream and the camel-toolbox apps, but that implies more complexity for people just trying to get something working.

Comment (birkland) [inline fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java]: @acoburn Maybe `ProcessorUtils` in `fcrepo-camel`?   Would it be useful for people building their own routes that need to filter stuff similarly?   If we don't want people using it approach, then it's probably fine to keep as-is.  If  moving it to fcrepo-camel does makes sense, that could probably be done later too under a separate issue

Comment (acoburn) [inline fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java]: Yes, that would be a good place for it to go, and now would be the time to put it in there. I'm ambivalent about it, but if you think that's a good approach, I'll do it.

Comment (birkland) [inline fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java]: How about just creating an issue for it now, sleep on it, and decide whether or not to go forward with that refactoring if the level of ambivalence changes.  I'm ambivalent too.  In the meantime, this PR looks great.  I wouldn't hold up merging for this.

Comment (acoburn) [inline fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java]: Ok, let's move forward with this PR and I'll add that to fcrepo-camel on Monday (this is exactly why I wanted to wait on the fcrepo-camel/4.5.0 release). We can track this issue here: https://jira.duraspace.org/browse/FCREPO-2345

**Pull request merged**